src/compas_ags/rhino/forceartist.py

```diff
@@ -92,24 +94,34 @@ def draw(self):
             color.update({edge: self.settings['color.edges:is_ind'] for edge in self.diagram.edges_where_dual({'is_ind': True})})
             # forces of the structure
             if self.settings['show.forces']:
-                color.update({edge: self.settings['color.tension'] for edge in self.diagram.edges_where_dual({'is_external': False}) if self.diagram.dual_edge_f(edge) > tol})
-                color.update({edge: self.settings['color.compression'] for edge in self.diagram.edges_where_dual({'is_external': False}) if self.diagram.dual_edge_f(edge) < -tol})
+                color.update(
+                    {edge: self.settings['color.tension'] for edge in self.diagram.edges_where_dual({'is_external': False}) if self.diagram.dual_edge_f(edge) > tol})
+                color.update(
+                    {edge: self.settings['color.compression'] for edge in self.diagram.edges_where_dual({'is_external': False}) if self.diagram.dual_edge_f(edge) < -tol})
             self.draw_edgelabels(text=text, color=color)
         # force magnitude labels
         if self.settings['show.forcelabels']:
             text = {}
             dual_edges = list(self.diagram.dual.edges())
-            for index, edge in enumerate(self.diagram.ordered_edges(self.diagram.dual)):
+            for index, (u, v) in enumerate(self.diagram.ordered_edges(self.diagram.dual)):
                 f = self.diagram.dual.edge_attribute(dual_edges[index], 'f')
-                text[edge] = "%s kN {%s}" % (round(f, 2), index)
+                if (u, v) in self.diagram.edges():
+                    text[(u, v)] = "%s kN {%s}" % (round(abs(f), 1), index)
+                else:
+                    text[(v, u)] = "%s kN {%s}" % (round(abs(f), 1), index)
```

Comment:
Do we like to show the edge label? I think we could remove it since we will probably work on a more robust update that could potentially remove the need to show the edge label all the time. so I would just do "%s kN"

Reply:
shall we keep it until that update happens?

Reply:
sounds good to me.